packages/fast-development-site-react/src/components/site/menu.tsx

```diff
@@ -28,56 +58,122 @@ const style: ComponentStyles<ISiteNavManagedClasses, IDevSiteDesignSystem> = {
         position: "absolute",
         background: "#FFFFFF",
         zIndex: "5",
-        boxShadow: `outline ${toPx(-2)} 0 ${toPx(4)} ${toPx(4)} rgba(0, 0, 0, 0.2)`,
-        "&[aria-hidden=\"true\"]": {
-            display: "none"
-        }
+        height: "calc(100vh)",
+        top: "0",
+        left: "0",
+        width: toPx(320),
+        animationDuration: "0.25s",
+        animationFillMode: "both",
+        transformOrigin: "50% 20%",
```

Comment:
this appears to animate from a central point, I would think this should animate from the top left corner otherwise the waffle will move as it is animating.

Reply:
Actually that was a product of me trying to emulate the https://www.office.com/ animation, but I was able to use the debugger and grab the animation off of the site. The animation does animate from the center so this can simply be removed. Thanks for pointing it out.